dizoo/dmc2gym/envs/dmc2gym_env.py

```diff
@@ -0,0 +1,169 @@
+
+
+def dmc2gym_reward_space(minimum=0, maximum=1) -> tuple:
+    return (
+        minimum,
+        maximum,
+    )
+
+
+dmc2gym_env_info = {
+    "ball_in_cup": {
+        "catch": {
+            "observation_space": dmc2gym_observation_space(8),
+            "state_space": dmc2gym_state_space(8),
+            "action_space": dmc2gym_action_space(2),
+            "reward_space": dmc2gym_reward_space()
+        }
+    },
+    "cartpole": {
+        "balance": {
+            "observation_space": dmc2gym_observation_space(8),
+            "state_space": dmc2gym_state_space(8),
+            "action_space": dmc2gym_action_space(1),
+            "reward_space": dmc2gym_reward_space()
+        },
+        "swingup": {
+            "observation_space": dmc2gym_observation_space(8),
+            "state_space": dmc2gym_state_space(8),
+            "action_space": dmc2gym_action_space(1),
+            "reward_space": dmc2gym_reward_space()
+        }
+    },
+    "cheetah": {
+        "run": {
+            "observation_space": dmc2gym_observation_space(17),
+            "state_space": dmc2gym_state_space(17),
+            "action_space": dmc2gym_action_space(6),
+            "reward_space": dmc2gym_reward_space()
+        }
+    },
+    "finger": {
+        "spin": {
+            "observation_space": dmc2gym_observation_space(9),
+            "state_space": dmc2gym_state_space(9),
+            "action_space": dmc2gym_action_space(1),
+            "reward_space": dmc2gym_reward_space()
+        }
+    },
+    "reacher": {
+        "easy": {
+            "observation_space": dmc2gym_observation_space(6),
+            "state_space": dmc2gym_state_space(6),
+            "action_space": dmc2gym_action_space(2),
+            "reward_space": dmc2gym_reward_space()
+        }
+    },
+    "walker": {
+        "walk": {
+            "observation_space": dmc2gym_observation_space(24),
+            "state_space": dmc2gym_state_space(24),
+            "action_space": dmc2gym_action_space(6),
+            "reward_space": dmc2gym_reward_space()
+        }
+    }
+}
+
+
+@ENV_REGISTRY.register('dmc2gym')
+class DMC2GymEnv(BaseEnv):
+
+    def __init__(self, cfg: dict = {}) -> None:
+        assert cfg.domain_name in dmc2gym_env_info
+        assert cfg.task_name in dmc2gym_env_info[cfg.domain_name]
+        self._domain_name = cfg.domain_name
+        self._task_name = cfg.task_name
+
+        self._cfg = cfg
+        self._init_flag = False
+
+        #to do
+        self._replay_path = None
+
+        self._observation_space = dmc2gym_env_info[cfg.domain_name][cfg.task_name]["observation_space"]
+        self._action_space = dmc2gym_env_info[cfg.domain_name][cfg.task_name]["action_space"]
+        self._reward_space = dmc2gym_env_info[cfg.domain_name][cfg.task_name]["reward_space"]
+
+    def reset(self) -> np.ndarray:
+        if not self._init_flag:
+            self._env = dmc2gym.make(domain_name=self._domain_name, task_name=self._task_name)
```

Comment:
you should add configuration like [this](https://github.com/MishaLaskin/curl/blob/master/train.py#L158)